README.md

```
@@ -25,24 +25,28 @@ Lien connects AI coding assistants like Cursor and Claude Code to your codebase
 
 ## Quick Start
 
+### Claude Code (recommended) — one-time plugin install
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)